js/main.js

```diff
@@ -1,17 +1,11860 @@
-/*!
- * Vue.js v2.6.9
- * (c) 2014-2019 Evan You
- * Released under the MIT License.
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
This seems to be the dev mode version of the file.
Running `npm run build` will build the production version of the file.

Reply:
Right – I'm always confused as to which commands to use and what to commit then. I'm on vacation now, so anyone is welcome to take over and add commits/amend, as also the "pressing Enter to open" has to be added.

Reply:
Okay. I'll generate the webpack files and see if I can add the enter to open functionality.